### ISSUE TITLE
Shortcuts

### ISSUE THREAD

**Issue by gdarai** [gdarai/SpaceRace#29]
One small idea... What abour repeated playability of this modpack?

Wouldn't it be just too tedious to do all the things again?
What is the slowest, most exhausting thing you guys did?

Do you think there should be some Shortcut option in the Guidebook? What Shortcuts would you like?

**Comment by gdarai** [comment]
I have one: Mystical Agriculture Flowers: This may be not a thing for the Shortcuts, but maybe it would be nice to have some simple recipe for the Mystical Bonemeal the same way we have one for normal Bone Meal. What do you think guys?
Or maybe through the Shortcuts... just maybe. The thing is, that MA Flowers grow unbelievably slowly. So, how to speed it up?

Also the next one is Bees... do we want to have them slow only or do we want to have faster way for replays? I think alternative is always good.

**Comment by OlorinCZ** [comment]
Hmm, how do you make a shortcut that does not count as a cheat mode? Maybe a special tab in the quest book?

Anyway I do not think mystical flowers are that much of an issue plus they can be sped up by their own means (watering cans, worms, lanterns).

The forestry on the othe hand is notoriously time consuming. If you consider adding shortcuts, this would definetly be something that needs them the most.

**Comment by gdarai** [comment]
@OlorinCZ Yes, I mean Questbook chapter. And yes that is cheating, but hey, we want it to be fun, not pain, right?

**Comment by kotja3** [comment]
I don't know if it belongs here, but all Mystical Agriculture essence doable with master infusion crystal (it could be made more expensive) would be nice.

**Comment by gdarai** [comment]
@kotja3 Yes, it belongs here... Well... the idea is, that if we don't want to do some tedious work, here can be a workAround.
I would not reactivate the powers of the master crystal since it is on purpous replaced, but...

Tier 1 - 3 are allready doable with the master crystal, right? The issue is the tedious tier 4 and 5.
So, I see two cheats... to give here as a cheat to player the master crystal OR to give here to player some even more powerfull crystal, which can do them all. (And preserve the master crystal since it kind of works, where it is.)

So far, I added cheats for lemon trees, metallic bees and sipiri saplings which does the same thing.
Adding the master crystal as an early cheat seems ok. I will think even more about it.

**Comment by kotja3** [comment]
Why sipiri sapling?

**Comment by gdarai** [comment]
Well... sipiri is the source for wood, necessary for some mystical agriculture high end crafting.
